### PR TITLE
Added brief info on how notifications gets delivered 

### PR DESCRIPTION
**Pull request by ivlists**
Update teams-admin-alerts.md

### CODE REVIEW

File: Teams/alerts/teams-admin-alerts.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You can use Teams monitoring and alerting to do the following items:
 - Take corrective actions to get things back on-track.
 
 > [!NOTE]
-> The alert functionality within the Teams admin center is not available in GCC/GCC-High environments.
+> The some of the alert functionalities within the Teams admin center is not available in GCC/GCC-High/DoD environments.
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:33]:
```suggestion
> Some of the alert functionalities within the Teams admin center are not available in GCC/GCC-High/DoD environments.
```

File: Teams/alerts/teams-admin-alerts.md [hunk 2]
```diff
@@ -40,17 +40,23 @@ You must be a global admin in Microsoft 365 or a Teams service admin to configur
 2. From the left navigation, select **Notifications & alerts**.
 3. Choose the rule you want to configure from **Rules**.
 
+## How alerts are delivered
+Alerts are delivered either as a Teams channel message or via webhooks depending on what options admin configures in rule.
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:44]:
```suggestion
Alerts are delivered either as a Teams channel message or via webhooks, depending on what options the admin configures in the rule.
```

File: Teams/alerts/teams-admin-alerts.md [hunk 3]
```diff
@@ -40,17 +40,23 @@ You must be a global admin in Microsoft 365 or a Teams service admin to configur
...
+
+Teams Channel alerts are delivered through bot app named **Teams Admin Center**. This bot app is auto-instlled into Team that is choosen by admin while rule configuration to recieve notifications.
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:46]:
```suggestion
Teams Channel alerts are delivered via a bot app named **Teams Admin Center**. This bot app is auto-installed into Teams and is chosen by the admin during rule configuration to receive notifications.   
```

File: Teams/alerts/teams-admin-alerts.md [hunk 4]
```diff
@@ -40,17 +40,23 @@ You must be a global admin in Microsoft 365 or a Teams service admin to configur
...
 The following is a list of the Teams monitoring rules available in the Teams admin center.
 
-|Rule  |Monitoring capability|What's monitored? |
-|---------|---------|---------|
-|[App submissions](../submit-approve-custom-apps.md) |Teams Apps | Proactively monitor Teams apps if they are submitted for approval.|
-|[Device state rule](device-health-status.md)  |Teams Devices | Proactively monitor Teams devices if they go offline.|
-|[Audio quality for in-progress meetings](alerts-in-progress-meeting-audio.md)|Teams meetings|Specified users' audio quality for in-progress meetings|
-|[Video quality for in-progress meetings](alerts-in-progress-meeting-video.md)|Teams meetings|Specified users' video quality for in-progress meetings|
-|[Screen sharing quality for in-progress meetings](alerts-in-progress-meeting-screen-sharing.md)|Teams meetings|Specified users' app sharing quality for in-progress meetings|
+|Rule  |Monitoring capability|What's monitored? |Public|GCC|GCCH|DoD|
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:53]:
```suggestion
|Rule  |Monitoring capability|What's monitored? |Public|GCC|GCC-H|DoD|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ivlists, kindly commit the suggested changes. Many thanks.

File: Teams/alerts/teams-admin-alerts.md [hunk 5]
```diff
@@ -40,17 +40,23 @@ You must be a global admin in Microsoft 365 or a Teams service admin to configur
 2. From the left navigation, select **Notifications & alerts**.
 3. Choose the rule you want to configure from **Rules**.
 
+## How alerts are delivered
+Alerts are delivered either as a Teams channel message or via webhooks depending on what options admin configures in rule.
+
+Teams Channel alerts are delivered through bot app named **Teams Admin Center**. This bot app is auto-instlled into Team that is choosen by admin while rule configuration to recieve notifications.   
+
+
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:48]:
```suggestion
```

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**